uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/request-modal/details/leave.html

```diff
@@ -5,13 +5,13 @@
       <div class="col-xs-8">
         <div class="btn-group btn-group-justified">
           <label
-            ng-disabled="$ctrl.isMode('view') || ($ctrl.isRole('manager') && !$ctrl.isLeaveType('sickness'))"
+            ng-disabled="$ctrl.isMode('view') || (!$ctrl.isMode('create') && !$ctrl.isLeaveType('sickness'))"
```

Comment:
This is the button which display the Multiple Days button, and it should only be disabled when the user is not in `create` mode regardless of the `role`. 
Previously the $ctrl.isRole('manager') was only true when the user is manager but not in create mode. As the $ctrl.isRole('manager') is fixed now, the !$ctrl.isMode('create') is necessary to be added otherwise the button will be disabled for Create mode for manager which we dont want.

Also Manager can edit everything if its a sickness request.